android/lib/src/main/java/com/ern/api/impl/navigation/views/MiniAppView.java

```diff
@@ -0,0 +1,78 @@
+package com.ern.api.impl.navigation.views;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.os.Build;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.AttributeSet;
+import android.widget.FrameLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
+
+import com.ern.api.impl.core.ElectrodeBaseFragmentDelegate;
+import com.facebook.react.ReactRootView;
+import com.walmartlabs.ern.navigation.R;
+
+public class MiniAppView extends FrameLayout {
+    public interface OnSetInitialPropsListener {
+        @Nullable
+        Bundle getProps(@NonNull String componentName);
+    }
+
+    private ReactRootView mReactRootView;
+    private ElectrodeBaseFragmentDelegate.ElectrodeActivityListener mReactDelegate;
```

Comment:
These two members can be converted to local variables.. Or do we need them here?

Reply:
Done, can move back if required in the future.